src/utils.js

```diff
@@ -407,15 +408,10 @@ module.exports = (function() {
       var flickrURL = url + "?" + queryString + signature;
 
       var req = request.post(flickrURL, function(error, response, body) {
-        // format:json does not actually work, so we need to grab the photo ID from the response XML:
-        // <?xml version="1.0" encoding="utf-8" ?>\n<rsp stat="ok">\n<photoid>.........</photoid>\n</rsp>\n
-        var data;
-        if(!body) {
-          error = error || "No body found in response";
-        } else if (body.indexOf('rsp stat="ok"')>-1) {
-          data = parseInt(body.split("<photoid>")[1].split("</photoid>")[0], 10);
-        }
-        callback(error, data);
+	      if(!body) return callback("No body found in response");
```

Comment:
looks like there's some indent issues (tabs vs. spaces) in these four lines - also could I ask you to use `{ }` around the returns, even though they're single lines? It makes it easier to extend (even if it's just console debug statements later)

Reply:
Indent issues: Oops, forgot this
{ } in return: Sure